gef-integration.target/gef-integration.target

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<?pde version="3.8"?><target name="gef-integration" sequenceNumber="1">
+<?pde version="3.8"?>
+<target name="gef-integration" sequenceNumber="1">
```

Comment:
Remove it or make it separate commit f.e. `gef-integration: fix .target formatting`